docs/Managing_Userids_and_Passwords.Rmd

```diff
@@ -208,7 +208,7 @@ next log-in.
 
 ## Have an issue?
 
-That's all for the tutorial. If you have any issues or feature requests as it pertains to managing user IDs and passwords in the app, please open a new issue on our [Github repo](https://github.com/pharmaR/risk_assessment/issues). Merry managing!
+That's all for the tutorial. If you have any issues or feature requests as it pertains to managing user IDs and passwords in the app, please open a new issue on our [Github repo](https://github.com/pharmaR/riskassessment/issues). Merry managing!
```

Comment:
The Managing_Userids_and_Passwords.html file was not updated with this change.

Reply:
@Jeff-Thompson12 fixed.